### PR TITLE
Automatically run `convox init` if needed when running `convox start`

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;os2MuWLr&#x2F;168-automatic-convox-init"> Automatic convox init</a>

File: api/manifest/manifest.go [hunk 1]
```diff
@@ -591,3 +651,112 @@ func randomString(size int) string {
 	}
 	return string(b)
 }
+
```
Comment (csquared) [inline api/manifest/manifest.go]: this file is getting kind of large - you may want to consider putting the detection stuff in one place

**Comment by csquared** [comment]
:+1:

**Comment by mattmanning** [comment]
:+1:

File: api/manifest/manifest.go (hunk 1 above)
Comment (ddollar) [inline api/manifest/manifest.go]: Refactor is coming

**Pull request merged**